Installers/Linux/RUN/postinstall.sh

```diff
@@ -88,6 +84,18 @@ case "$choice2" in
 	sudo -H -u $SUDO_USER bash -c 'mdtool setup install $DIR/Main/MonoDevelop.MonoGame.mpack'
 esac
 
+#MonoGame.Content.Builder.targets
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame
+fi
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame/v3.0 ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame/v3.0
+fi
+
+sudo cp $DIR/Main/MonoGame.Content.Builder.targets tmp/usr/lib/mono/xbuild/MonoGame/v3.0/
```

Comment:
I've only included Linux assemblies(since Linux can only build for MonoGame stuff for Linux) and also put them to be installed in "/usr/lib/mono/xbuild/MonoGame/v3.0/Assemblies/"